### PR TITLE
DIFF: Remove macros

### PR DESCRIPTION
**Pull request by g-back**
Ref #3170

### CODE REVIEW

File: src/utils/zcl_abapgit_diff.clas.abap [hunk 1]
```diff
@@ -158,42 +162,27 @@ CLASS ZCL_ABAPGIT_DIFF IMPLEMENTATION.
...
-    _add_regex '^\s*START-OF-'.
-    _add_regex '^\s*INITIALIZATION(\s|\.)'.
-
     LOOP AT mt_diff ASSIGNING <ls_diff>.
 
       CLEAR lv_offs.
       <ls_diff>-beacon = lv_beacon_idx.
 
-      LOOP AT lt_regex_set INTO lo_regex. "
+      LOOP AT create_regex_set( ) INTO lo_regex.
```
Comment (sbcgua) [inline src/utils/zcl_abapgit_diff.clas.abap]: 7.4 syntax ?

Comment (g-back) [reply src/utils/zcl_abapgit_diff.clas.abap]: Is it?  Wasn't sure, thought abaplint would catch. Will change!

Comment (larshp) [reply src/utils/zcl_abapgit_diff.clas.abap]: @sbcgua did you test it? in that case its a bug in abaplint, can you help opening an issue for it?

Comment (sbcgua) [reply src/utils/zcl_abapgit_diff.clas.abap]: Certainly does not work on 731

image

will open abaplint issue

Comment (sbcgua) [reply src/utils/zcl_abapgit_diff.clas.abap]: https://github.com/abaplint/abaplint/issues/768

**Pull request merged**